packages/runtime-core/src/componentEmits.ts

```diff
@@ -66,7 +90,8 @@ export type EmitFn<
           ? (event: key, ...args: Args) => void
           : (event: key, ...args: any[]) => void
       }[Event]
-    >
+    >) &
+  EmitVModelUpdate<P>
```

Comment:
Is this adding `emit('update:foo')` for every prop even if they don't have an emit declared for it?